packaging/vagrant/Vagrantfile

```diff
@@ -191,6 +216,21 @@ EOF2
     ch-run --version
   EOF
 
+  # Install ch-grow dependencies
+  c.vm.provision "ch-grow", type: "shell", privileged: true,
+                 inline: <<-EOF
+    yum -y install golang \
+                   go-md2man \
+                   python3.6 \
+                   python36-pip \
+                   skopeo
+
+    go get github.com/openSUSE/umoci
+    cd "/root/go/src/github.com/openSUSE/umoci" || exit 1
+    make VERSION=0.4.4 install
```

Comment:
What's the reasoning to build umoci from source? They have a nice binary download.

If we already have the Go toolchain from Buildah above, it's probably not much practical difference, but we should have a reason either way and a brief comment explaining it.